src/main/java/control/Control.java

```diff
@@ -1,165 +1,165 @@
-		return sonido;
-	}
-
-	public void setSonido(Sonido sonido) {
-		this.sonido = sonido;
-	}
-
-	public Vista getVista() {
-		return vista;
-	}
-
-	public void setVista(Vista vista) {
-		this.vista = vista;
-	}
-}
+package control;
+
+import dominio.Barril;
+import dominio.Personaje;
+import dominio.Tablero;
+import vista.BarrilGrafico;
+import vista.CorazonGrafico;
+import vista.FinalGrafico;
+import vista.InicioGrafico;
+import vista.LlamaGrafico;
+import vista.MonoGrafico;
+import vista.Sonido;
+import vista.Vista;
+import vista.Sonido.TipoSonido;
+
+public class Control {
+	private Pulsador pulsador;
+	private static Control INSTANCE = null;
+	private CorazonGrafico corazon;
+	private Sonido sonido;
+	private Vista vista;
+	private Sonido intro;
+	private FinalGrafico fin;
+
+	private Control() {
+	}
+
+	private synchronized static void createInstance() {
+		if (INSTANCE == null) {
+			INSTANCE = new Control();
+		}
+	}
+
+	public static Control getInstance() {
+		if (INSTANCE == null)
+			createInstance();
+		return INSTANCE;
+	}
+
+	public void start() {
+		java.awt.EventQueue.invokeLater(new Runnable() {
+			public void run() {
+				Tablero tablero = new Tablero(getInstance());
+				vista = new Vista(tablero);
+				vista.setVisible(true);
+				new Thread(new InicioGrafico()).start();
+				intro = new Sonido(TipoSonido.INTRO);
+				new Thread(intro).start();
+			}
+		});
+	}
+
+	public void nuevaPartida() {
+		try {
+			vista.setFin(false);
+			corazon.setRunning(false);
+			fin.setRunning(false);
+		} catch (Exception e) {	}
+		Tablero tablero = new Tablero(Control.getInstance());
+		this.vista.setTablero(tablero);
+		this.pulsador.setTablero(tablero);
+		new Thread(new Sonido(TipoSonido.COIN)).start();
+		vista.perderVida();
+		new Thread(tablero).start();
+	}
+
+	public void primeraPartida(Tablero tablero) {
+		intro.setLoop(false);
+		new Thread(new Sonido(TipoSonido.COIN)).start();
+		new Thread(tablero).start();
+		new Thread(new LlamaGrafico()).start();
+		pulsador = new Pulsador(tablero);
+		new Thread(pulsador).start();
+	}
+
+	public void labelWin() {
+		corazon = new CorazonGrafico(vista, true);
+		fin = new FinalGrafico(vista, true);
+		new Thread(corazon).start();
+		new Thread(fin).start();
+	}
+	
+	public void labelLose() {
+		corazon = new CorazonGrafico(vista, false);
+		fin = new FinalGrafico(vista, false);
+		new Thread(corazon).start();
+		new Thread(fin).start();
+	}
+	
+
+	public void lanzaBarril(Barril barril) {
+		new Thread(new BarrilGrafico(barril, getInstance())).start();
+	}
+
+	public void movimientoMono() {
+		new Thread(new MonoGrafico(getInstance())).start();
+	}
+
+	public void mataAMario() {
+		vista.vistaMuerto();
+		vista.perderVida();
+	}
+
+	public Pulsador getPulsador() {
+		return pulsador;
+	}
+
+	public void setPulsador(Pulsador pulsador) {
+		this.pulsador = pulsador;
+	}
+
+	public void saltoMario() {
+		vista.salto();
+		((Personaje) vista.getTablero().getElementos().get(0)).setSaltando(false);
```

Comment:
deberías implementar un método en Tablero que sea getPersonaje (mejoráis la encapsulación y evitáis el acoplamiento)
Además deberías implementar una clase `PersonajeController` que exponga el metodo `salta()` (que llame a la instancia de su personaje con `personaje.setSaltando(false)` (mejoráis la encapsulación y evitáis el acoplamiento)